src/main/java/de/sakpaas/backend/service/OccupancyService.java

```diff
@@ -163,6 +166,15 @@ public void checkReportLimit(Location location, byte[] requestHash)
     }
   }
 
+  /**
+   * Returns True if the Date is a Public Holiday.
+   *
+   * @param date The Date
+   * @return Result if it is a public holiday
+   */
+  private boolean checkForPublicHoliday(Date date) {
```

Comment:
`isPublicHoliday()`?

Reply:
`checkFor...()` klingt immer so als würde die Methode selbst irgendetwas auslösen aber nichts zurück geben. In Java wird für `boolean`s eigentlich immer `is...()` oder `has...()` verwendet.

Reply:
Done